packages/synapse-react-client/src/components/IconSvg/IconSvg.tsx

```diff
@@ -235,6 +237,8 @@ export const IconStrings = [
   'createVersion',
   'email',
   'addConditions',
+  'sortUp',
+  'sortDown',
```

Comment:
Added `sortUp` and `sortDown` to IconSvg. These are the same icons as seen in SynapseTable today, but many of the icons that SynapseTable uses use older Icon components that we are not actively investing in as much as this `IconSvg`